src/metadatas.rs

```diff
@@ -144,6 +152,35 @@ pub fn extract_metadata(gtfs: &gtfs_structures::RawGtfs) -> Metadata {
     }
 }
 
+impl Metadata {
```

Comment:
A RawGtfs is not enough to compute this information, we need a proper Gtfs. Hence this method and the fact that `stops_with_wheelchair_info_count` is an `Option<usize>`, in case the Gtfs cannot be loaded.